src/transform/FooterContainer.js

```diff
@@ -22,11 +22,12 @@ const updateBottomPaddingToAllowReadMoreToScrollToTop = window => {
  * @return {void}
  */
 const updateLeftAndRightMargin = (margin, document) => {
-  const elements = Polyfill.querySelectorAll(document, `
-    #pagelib_footer_container_menu_heading,
-    #pagelib_footer_container_readmore,
-    #pagelib_footer_container_legal
-  `)
+  const selectors = [
```

Comment:
In the js file produced by babel I noticed it had '\n' inside of this string and it just seemed odd. Didn't appear to affect functionality, but thought it was easy to remove those.